### ISSUE TITLE
Error in building from the project.

### ISSUE THREAD

**Issue by kishorej88** [LDClark/PlanCheck#6]
Dear Sir ,

              At first, I want to say thank you for sharing such interesting work as open source.  I am very new to scripting while building the script from your project I am facing the error below in visual studio. Can your please help me to resolve that?  I am using eclipse version 16.1 is that cause this problem or i am missing some .dll.

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0006	Metadata file 'C:\Users\91986\Downloads\PlanCheck-master\PlanCheck-master\PlanCheck.Script\bin\x64\Debug\PlanCheck-2.0.0.0.esapi.dll' could not be found	PlanCheck.Runner	C:\Users\91986\Downloads\PlanCheck-master\PlanCheck-master\PlanCheck.Runner\CSC	1	Active

**Comment by LDClark** [comment]
Do you get that when you do a Rebuild All?  Did you install the Nuget package Esapi Essentials and set the runner as the startup project?  Thanks for trying it out!

**Comment by kishorej88** [comment]
Thank you sir for your prompt reply . What i did is mentioned below in the steps if something i am missing please guide me .

1. Downloaded zip repository from GitHub.
2. Extracted data from zip folder .
3. Opened PlanCheck.sln in Microsoft visual basic 2019 .
4. Remove references to VMS.TPS.Common.Model.API and VMS.TPS.Common.Model.Types, and add the local dlls from my eclipse (V16.1) .
5. Right-click solution > Restore NuGet Packages
6. Then i try to rebuild the solution.

As i am very new to scripting and visual basics and maybe missing something please let me know if you can found the fault .

Error1: Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0234	The type or namespace name 'Common' does not exist in the namespace 'VMS.TPS' (are you missing an assembly reference?)	PlanCheck	C:\Users\91986\Desktop\New folder\PlanCheck-master\PlanCheck.Script\ViewModels\PQMSummaryViewModel.cs	7	Active


Error 2: Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0006	Metadata file 'C:\Users\91986\Desktop\New folder\PlanCheck-master\PlanCheck.Script\bin\x64\Debug\PlanCheck-2.0.0.0.esapi.dll' could not be found	PlanCheck.Runner	C:\Users\91986\Desktop\New folder\PlanCheck-master\PlanCheck.Runner\CSC	1	Active

Thank you

**Comment by LDClark** [comment]
I haven't used Plancheck with 16.1 yet, that could be the issue.  You might have luck looking though the Output menu for specific exceptions and see what you have to modify in order for it to build.  Sometimes the methods change slightly within ESAPI on the VMS.TPS side and that could be all that needs to be done.  Visual Studio should give you a red squiggly line to indicate that some classes need updating (it looks like there is an issue under \ViewModels\PQMSummaryViewModel.cs).  

I hope to update it soon.  Ideally there would be if/else lines around the 13.6/13.7/15.5/15.6/16.1 conditional methods so that it would work for all VMS.TPS references, but right now it most likely only works for version 15.  I'll try to get access to 16.1 dlls to see if I can fix it from my end.

**Comment by kishorej88** [comment]
thank you for your reply. The error i am getting is 
Error CS0006:Metadata file 'C:\Users\91986\sktop\New folder\PlanCheck-master\PlanCheck.Script\bin\x64\Debug\PlanCheck-2.0.0.0.esapi.dll' could not be found. what is this dll  and how can I resolve this. If you ok i  can share my 16.1 dlls.

**Comment by LDClark** [comment]
I was able to get the 16.1 dlls from the Varian Innovation Center website (I just signed up, but has been helpful so far).

I was able to get it to build by doing:
-Replace the 15.6 dlls with the 16.1 version in the two projects PlanCheck and PlanCheck.Runner. 
-Update the Target Framework to 4.6.1 in each project PlanCheck, PlanCheck.Reporting, PlanCheck.Reporting.MigraDoc, and PlanCheck.Runner (right-click to get application properties)
-Build>Rebuild Solution

Let me know if that doesn't work, and sorry for taking so long to get back to you if you haven't already figured it out.

Landon
